source/config_test.go

```diff
@@ -25,43 +25,44 @@ func TestConfig_Validate(t *testing.T) {
 		name    string
 		cfg     Config
 		wantErr bool
-	}{{
-		name: "valid config",
-		cfg: Config{
-			URL:     "postgresql://meroxauser:meroxapass@127.0.0.1:5432/meroxadb",
-			Table:   []string{"table1", "table2"},
-			Key:     []string{"table1:key1"},
-			CDCMode: CDCModeLogrepl,
+	}{
+		{
```

Comment:
Any specific reason you modified this formatting? Seems like the linter doesn't complain about it and neither do the formatting tools..

Reply:
ah, because of gofumpt...

Reply:
Yeah I figured, no problem, just curious 😉